datafusion/physical-expr/src/aggregate/bit_and_or_xor.rs

```diff
@@ -279,35 +280,31 @@ impl AggregateExpr for BitAnd {
         use std::ops::BitAndAssign;
         match self.data_type {
             DataType::Int8 => {
-                instantiate_primitive_accumulator!(self, Int8Type, |x, y| x
-                    .bitand_assign(y))
+                instantiate_accumulator!(self, -1, Int8Type, |x, y| x.bitand_assign(y))
```

Comment:
the fix is passing in `MAX` / `-1` here to get a bitpattern of all `1` initially

Reply:
The fix looks good. It's better to leave a comment like above in the code.

Reply:
Comment added in  https://github.com/apache/arrow-datafusion/pull/6964